### ISSUE TITLE
force Boris to deliver intro text

### ISSUE THREAD

**Issue by davidwhogg** [davidwhogg/InferenceOfVariances#3]
cc @ixkael

**Comment by ixkael** [comment]
thanks for the reminder and sorry about the delay. I made a couple of attempts over the summer but I had to do some extra background reading to convince myself that there are no overwhelmingly strong arguments in this paper. But I'll ignore this for now and add the intro text I promised this week!

**Comment by davidwhogg** [comment]
"No overwhelmingly strong arguments"... Meaning what...? I am happy to reconsider at any level; this is a side project for me!

**Comment by ixkael** [comment]
As Eisenstein said in an old email you forwarded to me, what you describe in the paper is formally correct but the quantitative consequences are unclear. In current LSS analyses modes are usually binned and gaussianized ; for the low-ell CMB, improved likelihoods have been developed. So my only objection is about the claim that using the pseudo-likelihood (gaussian) is a big issue for current LSS data sets at BAO scale, since it is not demonstrated here. What do you think? Did you get more feedback from Eisenstein and others? In any case I'm adding references and background to the intro now.

**Comment by davidwhogg** [comment]
Okay good I understand now. I do still think a correct approach is useful. Do you know of any project that does LSS correctly at the BAO scale? If so, let's look. If not, then let's demonstrate the inevitable wrongness. I think I know how to do this.

**Comment by ixkael** [comment]
Yes, this paper is making a point about the correctness of the approach and this is very valuable. It would be useful to gain some quantitative intuition about LSS specific problems, though. I don't know of any papers that use the correct likelihood, especially on data. Not mentioning the fact that the covariance of the correlation function is also estimated from mocks! Even the pseudo-likelihood has to be modified, see https://arxiv.org/abs/1609.00504, so ABC would be even more useful there (emphasizing this could make the paper even stronger).

I would say that for current analyses, the number of modes measured and averaged (in each bin and in the total correlation function) must make this effect negligible compared to other sources of errors. We could probably quantify this with a simple simulation, with BOSS-like numbers, to see if the pseudo-likelihood is a good approximation.
